fields/types/list/ListFilter.js

```diff
@@ -0,0 +1,43 @@
+/*
+TODO: Not sure what this should look like yet,
```

Comment:
I was thinking that nested components should show as indented filters, and they would simply have the filters of their subcomponents. So for an array of Locations you'd be able to filter on street, and you'd find all documents with that street as one of the array values.

That means, you're doing "list contains". Other options would be "only contains", "does not contain", "has at least/at most/exactly x items", …